test/python/tests_linux_gpu.txt

```diff
@@ -0,0 +1,42 @@
+# Manifest to specify included & excluded tests
+# Comment lines start with #
+# This manifest is consumed by nGraph-Bridge PyTest framework via .../test/python/conftest.py
+
+# Read comments in tests_common.txt for allowed regex/patterns
+
+###################################################
+[IMPORT]
+# Read in one/more external manifest file(s)
+# Path specified is relative to this file's path
+
+tests_linux_cpu.txt
```

Comment:
Shouldn't the import be for the common.txt?

Reply:
Reusing rules as in CPU, and then adjust the extra skips

Reply:
No, why? The same inclusions/exclusions as for CPU apply in the case of GPU.

Reply:
Actually I agree with @kanvi-nervana. In both of these cases there are too many exclusions that it may be better to base this off of `common.txt`.

Reply:
If you import CPU.txt then it's exclusions will also be carried forward, right? Those exclusions might not be needed for GPU and you'll then explicitly put them in the RUN list which is not required.

Reply:
Correct. Can we enable more tests as part of a separate PR? As of this PR, we will have a baseline of equivalent to CPU minus few.

Reply:
It's up to you, but we are in agreement that the base for every pipeline should be common.txt

Reply:
More tests can be enabled as a part of a separate PR (since it's a new change) but if we agree that GPU tests shouldn't depend on the CPU tests then that change should be made as a part of this PR.

I understand that the CPU pipeline is the most permissive but at the risk of being slightly repetitive, I think it's generally a better idea to keep them separate from the outset?